src/test/java/seedu/address/storage/JsonAdaptedAppointmentTest.java

```diff
@@ -45,8 +45,8 @@ void toModelType_validAppointmentDetails_returnsAppointment() throws Exception {
         assertEquals(VALID_APPOINTMENT_TYPE, modelAppointment.getAppointmentType().value);
         assertEquals(LocalDateTime.parse(VALID_APPOINTMENT_DATE_TIME), modelAppointment.getAppointmentDateTime());
         assertEquals(VALID_PERSON_ID, modelAppointment.getPerson().getPersonId());
-        assertEquals(VALID_SICKNESS, modelAppointment.getSickness().value);
-        assertEquals(VALID_MEDICINE, modelAppointment.getMedicine().value);
+        assertEquals(VALID_SICKNESS, modelAppointment.getSickness().get().value);
+        assertEquals(VALID_MEDICINE, modelAppointment.getMedicine().get().value);
```

Comment:
Great job editing the current test cases so that they fit the Optional Class as well!